web/src/styles/global.css

```
@@ -1019,6 +1019,352 @@ body {
+  font-size: 14px;
+  line-height: 1.45;
+}
+.eac-field-label {
+  margin: 0;
+  color: var(--text-secondary);
+  font-weight: 500;
+}
+.eac-field-value {
+  margin: 0;
+  color: var(--text);
+  word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)